docs/development.rst

```diff
@@ -203,6 +191,42 @@ Then execute the following and copy the markdown version for pasting to GitHub r
 
 After the release commit has been pushed and a release has been tagged, set a development version in the same above files. This is basically the next minor release postfixed by ``-dev``.
 
+Developing with Docker
+------------------------
+
+If you choose, you may develop Socialhome using Docker, rather than installing Postfix and Redis manually on your computer.
+
+
+Supported versions
+..................
+
+This guide assumes you are running Docker on a GNU/Linux based system such as Ubuntu, Debian or Fedora Linux. It may be possible to run this on other platforms where Docker is supported, but those are untested.
+
+The docker development installation was tested on Docker version 17.09 and docker-compose 1.16.1.
+
+Steps
+......
+
+The first step is to copy the ``docker/dev/docker-compose.yml`` file to the root of the project. eg
```

Comment:
The file is missing from this commit, probably need to tune the `.gitignore` to exlude it from that path, but include it when in root path.

Reply:
`docker-compose.yml` is still missing from the PR.

Reply:
It's on the .gitignore and in the docker/dev directory. :heavy_check_mark: